01/prvocislo.py

```diff
@@ -0,0 +1,32 @@
+from math import sqrt
+import sys
+
+# one of larger prime numbers is f.e. 10657331232548839
+
+def is_prime_number(num):
+    if num%2 ==0 and num>2 or num==1:				
+       return False
+    # algorithm for odd numbers    
+    else:
+        for i in range(3,round(sqrt(user_num))+1,2):
+            if user_num%i ==0:
+               return False
+        return True
+    
+
+try:
+    user_num = int(sys.argv[1])
+except ValueError:
+    print(f"Zadané číslo musí být přirozené.")
```

Comment:
```suggestion
    print("Zadané číslo musí být celé.")
```
Pokud bys chtěla neuznávat nepřirozená celá čísla tak jak to máš teď tak neaplikuj.